docs/application-connector/05-01-application-operator.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Application Operator sub-chart
+type: Configuration
+---
+
+To configure the Application Operator sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **controller.args.installationTimeout** | Specifies a period of time provided for Application Gateway and Event Service installation. The services are required for an Application to be operational. The value is provided in seconds.| `240` |
```

Comment:
There is also ApplicationConnectivityValidator installed. Sorry, I missed that :(. Please see the diagram: https://github.com/kyma-project/kyma/blob/master/docs/application-connector/02-01-application-connector.md